.github/ISSUE_TEMPLATE/ask-question.yml

```diff
@@ -0,0 +1,50 @@
+name: 🙋‍♂️ Ask a question
+description: Tell us what's on your mind
+title: "[question]: "
+labels: ["triage"]
+# assignees:
+#   - assign your sdk team e.g., onesignal/unity
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Having issues integrating this SDK?
+  - type: textarea
+    id: question
+    attributes:
+      label: How can we help?
+      description: Specific question regarding integrating this SDK.
+      placeholder: How do I...?
+    validations:
+      required: true
+  - type: textarea
+    id: expo-sdk-version
+    attributes:
+      label: OneSignal Expo SDK version
+      description: The version of the OneSignal Expo SDK you have a question for.
+      placeholder: 1.0.0
+    validations:
+      required: true
+  - type: dropdown
+    id: platform
+    attributes:
+      label: Platform
+      description: Which platform are you having an issue with?
+      multiple: true
+      options:
```

Comment:
I think this list should include web platforms, not mobile unless we want to support OneSignal+Vue apps not working as expected on mobile (this is outside of my area; please go with whatever the SDK suggests).